persistence/module.go

```diff
@@ -239,7 +239,7 @@ func (m *persistenceModule) GetTxIndexer() indexer.TxIndexer {
 }
 
 func (m *persistenceModule) GetTreeStore() modules.TreeStoreModule {
-	return m.stateTrees
+	return m.treeStore
 }
```

Comment:
This method becomes redundant now that `treeStore` is dependency injectable.

Consumers of this method should retrieve it from the bus/registry e.g. `bus.GetTreeStore()`. Until #880 is resolved you won't be able to implement `Bus#GetTreeStore()`, which is the same thing that happened in the `PeerstoreProvider` submodule refactor (see: #895).

You can work around this in a similar way, using an unexported temporary helper with does the `bus.GetModuleRegistry().GetModule(...)` call. See [`p2p#getPeerstoreProvider`](https://github.com/pokt-network/pocket/blob/refactor/p2p-submodules/p2p/module.go#L530) as an example.